x-pack/plugins/search_playground/public/components/edit_context/edit_context_flyout.tsx

```diff
@@ -62,13 +64,23 @@ export const EditContextFlyout: React.FC<EditContextFlyoutProps> = ({ onClose })
       ...tempSourceFields,
       [index]: f.filter(({ checked }) => checked === 'on').map(({ label }) => label),
     });
+    usageTracker.click('edit_context_field_toggled');
   };
 
   const saveSourceFields = () => {
+    usageTracker.click('edit_context_save');
     onChangeSourceFields(tempSourceFields);
     onChangeSize(docSize);
     onClose();
   };
+  const handleDocSizeChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    usageTracker.click('edit_context_doc_size_changed');
+    setDocSize(Number(e.target.value));
+  };
+
+  useEffect(() => {
+    usageTracker.load('edit_context_flyout_opened');
```

Comment:
Are we planning to catch any action performed by the user?

Reply:
To get rid of unclaimed functions and give more attention to what users need. So better to store more data